bootloader/utility.h

```diff
@@ -3,6 +3,7 @@
 
 namespace Utility {
   extern int versionSum(const char * version, int length);
```

Comment:
Can you remove `versionSum` from here and in the source, since it doesn't work, and is not used